### PR TITLE
Frontend : Create Html_Accessibility.md

### PR DESCRIPTION
**Pull request by Pranshu321**
<hr>

## Description 📜

In this commit I have created Html_Accessibility file which contain full detail explanation of topic accessibility in this i cover all the topic which this topic can contain and i also give code snippets and examples that explain the text in great detail.
Issue no. Closed : #2820 
<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] Frontend Dev HTML/CSS/JS

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: Web_Development/FrontEnd/Html_Accessibility.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# HTML Accessibility
+This provised the user to navigate and interact in  a better way with your website . That's make your HTML code as <b>senmantic</b> as possible.
```
Comment (padmajabhol) [inline Web_Development/FrontEnd/Html_Accessibility.md:2]: grammatical error

File: Web_Development/FrontEnd/Html_Accessibility.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+## Semantic HTML
+These are the elements which have a meaning or we can say they clearly define about the content which they holding.
```
Comment (padmajabhol) [inline Web_Development/FrontEnd/Html_Accessibility.md:5]: Here too.

**Review by padmajabhol — COMMENTED** [review]
Lots of grammatical errors, please correct it.

File: Web_Development/FrontEnd/Html_Accessibility.md (hunk 2 above)
Comment (Pranshu321) [reply Web_Development/FrontEnd/Html_Accessibility.md]: Done mam now you can check and able to merge also @padmajabhol .

Comment (Pranshu321) [reply Web_Development/FrontEnd/Html_Accessibility.md]: Please review It again

**Review by ipsitagoel — CHANGES_REQUESTED** [review]
@Pranshu321 Make the changes as mentioned by @padmajabhol

File: Web_Development/FrontEnd/Html_Accessibility.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
+# HTML Accessibility
+This provides the user to navigate and interact in a better way with your website . That's why make your HTML code as semantic as possible.
```
Comment (ipsitagoel) [inline Web_Development/FrontEnd/Html_Accessibility.md:2]: @Pranshu321 Try to frame this in a better way so that it is more understandable.

Comment (Pranshu321) [reply Web_Development/FrontEnd/Html_Accessibility.md:2]: Ok

File: Web_Development/FrontEnd/Html_Accessibility.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+## Semantic HTML
+Semantic elements are the elements which have a meaning or we can say they clearly define about the content which they are holding.
+```
```
Comment (meinhoonharsh) [inline Web_Development/FrontEnd/Html_Accessibility.md:6]: You can use HTML Syntax Highlighting here

File: Web_Development/FrontEnd/Html_Accessibility.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+<form> </form> 
+ 
+// same this is tell us that it is used to make table.
+
+ <table> </table>
+```
+## Non-Semantic HTML
+They are elements which tell nothing about the content which they hold.
+```
```
Comment (meinhoonharsh) [inline Web_Development/FrontEnd/Html_Accessibility.md:17]: same goes with all other code snippets

File: Web_Development/FrontEnd/Html_Accessibility.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+## Alternative Text
+This tag used when we have to provide the alternate text for an image. The text in this tag will be showed when the image can't be displayed due to any kind of error in src attribute.
+```
+<img src="GWOC'21.jpg" alt="Logo of GWOC'21">
+```
+## Declaring The Langauge
+This is used inside the html tag , this is to declare the language of the web page. This is meaningful to assist search engines and browsers.
+```
+<!DOCTYPE html>
+<html lang="en>
```
Comment (meinhoonharsh) [inline Web_Development/FrontEnd/Html_Accessibility.md:43]: string opened but not closed

**Review by meinhoonharsh — CHANGES_REQUESTED** [review]
It's pretty nice, just need some Improvements

**Review by heet-kumar — CHANGES_REQUESTED** [review]
Also try to add references from where you have referred and prepared this documentation.

File: Web_Development/FrontEnd/Html_Accessibility.md [hunk 7]
```diff
@@ -38,12 +38,12 @@ This tag used when we have to provide the alternate text for an image. The text
 ```
 ## Declaring The Langauge
 This is used inside the html tag , this is to declare the language of the web page. This is meaningful to assist search engines and browsers.
-```
+```html
 <!DOCTYPE html>
 <html lang="en>
```
Comment (meinhoonharsh) [inline Web_Development/FrontEnd/Html_Accessibility.md]: `lang="en"` instead of `lang="en`

**Review by meinhoonharsh — CHANGES_REQUESTED** [review]

**Comment by code-reaper08** [comment]
> Good job @Pranshu321 @code-reaper08 Have a look.

LGTM ✨ 
You can merge it @ipsitagoel

**Pull request merged**